Cargo.toml

```diff
@@ -11,4 +11,5 @@ homepage = "https://github.com/danieldk/oxidized-transformers"
 [dependencies]
 candle-core = "0.3"
 candle-nn = "0.3"
+snafu = "0.8"
 thiserror = "1"
```

Comment:
Do we still need the `thiserror` dependency?

Reply:
Fixed.